nominatim/tools/database_import.py

```diff
@@ -38,6 +40,20 @@ def _require_version(module: str, actual: Tuple[int, int], expected: Tuple[int,
         raise UsageError(f'{module} is too old.')
 
 
+def check_existing_database_plugins(dsn: str):
+    """ Check that the database has the required plugins installed."""
+    with connect(dsn) as conn:
+        _require_version('PostgreSQL server',
+                         conn.server_version_tuple(),
+                         POSTGRESQL_REQUIRED_VERSION)
+        _require_version('PostGIS',
+                         conn.postgis_version_tuple(),
+                         POSTGIS_REQUIRED_VERSION)
+        _require_version('hstore',
```

Comment:
I don't think we require any specific minimal version of hstore. Simply check that the extension is loaded.